lib/adal.js

```diff
@@ -810,6 +843,9 @@ var AuthenticationContext = (function () {
                 this.verbose('Window is redirecting');
                 callback = this.callback;
             }
+            callback = this.callback;
+            if (callback)
+                callback(this._getItem(this.CONSTANTS.STORAGE.ERROR_DESCRIPTION), requestInfo.parameters[this.CONSTANTS.ACCESS_TOKEN] || requestInfo.parameters[this.CONSTANTS.ID_TOKEN]);
```

Comment:
Related to above comment. This callback is for login request and I think we should move it to the else if block. Also, we should only pass id_token to the callback in this case, no access_token.